oddt/fingerprints.py

```diff
@@ -528,6 +530,30 @@ def combinatorial_rmsd(reference, query):
         return np.sqrt((numla / nula) * (numpa / nupa))
 
 
+def PLECFP(ligand, protein, distance_cutoff=4.5, depth_protein=4, depth_ligand=2, fold_size=16384):
```

Comment:
Also you swap the order of the arguments - there is `ligand, protein` and then `depth_protein=4, depth_ligand=2`